src/System.Text.Json/src/System/Text/Json/Document/JsonElement.cs

```diff
@@ -972,6 +975,103 @@ internal string GetPropertyRawText()
             return _parent.GetPropertyRawValueAsString(_idx);
         }
 
+        /// <summary>
+        ///   Compares <paramref name="text" /> to the string value of this element.
+        /// </summary>
+        /// <param name="text">The text to compare against.</param>
+        /// <returns>
+        ///   <see langword="true" /> if the string value of this element matches <paramref name="text"/>,
+        ///   <see langword="false" /> otherwise.
+        /// </returns>
+        /// <exception cref="InvalidOperationException">
+        ///   This value's <see cref="Type"/> is not <see cref="JsonValueType.String"/> or <see cref="JsonTokenType.PropertyName"/>.
+        /// </exception>
+        /// <remarks>
+        ///   This method is functionally equal to doing an ordinal comparision of <paramref name="text" /> and
```

Comment:
nit: typo `comparision`

```suggestion
        ///   This method is functionally equal to doing an ordinal comparison of <paramref name="text" /> and
```